.github/workflows/npmpublish.yml

```diff
@@ -5,7 +5,7 @@ name: Node.js Package
 
 on:
   release:
-    types: [created]
+    types: [published]
```

Comment:
Changed to `published` since a draft release has a "Publish" button and a published event seems more logical.

![image](https://user-images.githubusercontent.com/44813512/80324318-34201600-87e5-11ea-88ed-abf0ac4257e4.png)